tools/proofers/srchrep.php

```diff
@@ -132,22 +99,7 @@ function preg_quote(str)
 </center>
 </form>
 <p><?php echo _("Warning: Undo is only possible for the most recent replace!"); ?></p>
-<p id='regex_help_title' onclick='toggle_regex_help();'><span id='regex_arrow'>&#9654;</span>
-<?php echo _('Regular Expression?'); ?></p>
-<p id='regex_help'><?php
-// TRANSLATORS: Description of the . character in a regualar expression
-echo ". &mdash; " . _("any character") . "<br>\n";
-// TRANSLATORS: Description of the [a-z0-9] class in a regular expression
-echo "[a-z0-9] &mdash; " . _("lowercase letters and numbers") . "<br>\n";
-// TRANSLATORS: Description of a{4} in a regular expression for an example
-echo "a{4} &mdash; " . _("four lowercase As") . "<br>\n";
-// TRANSLATORS: Description of [Aa]{6} in a regular expression for an example
-echo "[Aa]{6} &mdash; " . _("six As of either case") . "<br>\n";
-// TRANSLATORS: Description of A{2,8} in a regular expression for an example
-echo "A{2,8} &mdash; " . _("between 2 and 8 capital As") . "<br>\n";
-// TRANSLATORS: Description of [hb]e in a regular expression for an example
-echo "[hb]e &mdash; " . _("'he' or 'be'") . "<br>\n";
-?></p>
+<a href='<?php echo "$code_url/faq/prooffacehelp.php"; ?>' target='helpNewWin'><?php echo _('Regular Expression?'); ?></a>
```

Comment:
I realize you reused the prior text, but the link might be clearer if it was labeled 'Regular Expression Help'. @srjfoo, what do you think?

Reply:
I agree.